docs/setup/install/ingress-setup.md

```diff
@@ -115,6 +112,63 @@ spec:
               number: 80
         path: /grafana
         pathType: ImplementationSpecific  
+```
+
+## Enable HTTPS For Devtron
+
+Once ingress setup for devtron is done and you want to run Devtron over `https`, you need to add different annotations for different ingress controllers and load balancers.
+
+### 1. Nginx Ingress Controller
+
+In case of `nginx ingress controller`, add following annotations under `service.annotations` under nginx ingress controller to run devtron over `https`.
+
+(i) Amazon Web Services (AWS)
+
+If you are using AWS cloud, add the following annotations under `service.annotations` under nginx ingress controller.
+
+```bash
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "<acm-arn-here>"
+```
+
+(ii) Digital Ocean
+
+If you are using Digital Ocean cloud, add the following annotations under `service.annotations` under nginx ingress controller.
+
+```bash
+annotations:
+  service.beta.kubernetes.io/do-loadbalancer-protocol: "http"
+  service.beta.kubernetes.io/do-loadbalancer-tls-ports: "443"
+  service.beta.kubernetes.io/do-loadbalancer-certificate-id: "<your-certificate-id>"
+  service.beta.kubernetes.io/do-loadbalancer-redirect-http-to-https: "true"
+```
+
+### 2. Application Load Balancer (ALB)
+
+In case of AWS application load balancer, add following annotations under `ingress.annotations` to run devtron over `https`.
+
+```bash
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
+    alb.ingress.kubernetes.io/certificate-arn: "<acm-arn-here>"
+```
+
+### 3. Azure Application Gateway
+
+In case of AWS application load balancer, the following annotations need to be added under `ingress.annotations` to run devtron over `https`.
+
+```bash
+ annotations:
+  kubernetes.io/ingress.class: "azure/application-gateway"
+  appgw.ingress.kubernetes.io/backend-protocol: "http"
+  appgw.ingress.kubernetes.io/ssl-redirect: "true"
+  appgw.ingress.kubernetes.io/appgw-ssl-certificate: "<name-of-appgw-installed-certificate>"
+```
+> Note: Make sure to not use port 80 with HTTPS and port 443 with HTTP on the Pods.
```

Comment:
Note: Make sure NOT to use port 80 with HTTPS and port 443 with HTTP on the Pods.